code/modules/mob/living/simple_animal/abnormality/waw/slitcurrent.dm

```diff
@@ -0,0 +1,240 @@
+//Dream-Devouring Slitcurrent would be a breach that forces you to either try to kill it from it hitting the tubes or using the tubes to make you last longer.
+//When out you're on a timer due to everyone on the floor it's on taking oxygen damage.
+
+/mob/living/simple_animal/hostile/abnormality/slitcurrent
+	name = "\proper Dream-Devouring Slitcurrent"
+	desc = "An abnormality resembling a giant black and teal shark or fish. \
+	There's teal light tubes embedded in its body,"
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "slitcurrent"
+	icon_living = "slitcurrent"
+	pixel_x = -16
+	base_pixel_x = -16
+	move_to_delay = 3
+	rapid_melee = 2
+	melee_damage_lower = 25
+	melee_damage_upper = 35
+	melee_damage_type = RED_DAMAGE
+	maxHealth = 3000
+	health = 3000
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.5, WHITE_DAMAGE = 0.5, BLACK_DAMAGE = 0.5, PALE_DAMAGE = 0.5)
```

Comment:
Brute = 1 isnt needed in this list anymore.

Reply:
did it